cinema/serializers.py

```diff
@@ -132,6 +149,30 @@ def create(self, validated_data):
                 Ticket.objects.create(order=order, **ticket_data)
             return order
 
+    def update(self, instance, validated_data):
+        tickets_data = validated_data.get("tickets", [])
+        instance = super().update(instance, validated_data)
```

Comment:
In the `update` method, the `validated_data` is being used to update the instance before processing the tickets. This might lead to partial updates if the ticket processing fails. Consider moving the instance update after processing the tickets to ensure atomicity.